dev/packages/example/nginx-1.2.0/dataset/stubstatus/manifest.yml

```diff
@@ -12,7 +12,10 @@ release: beta
 
 streams:
   - input: "nginx/metrics"
-
-    # Only the variables have to be repeated that are not specified as part of the input
+    title: Nginx stubstatus metrics
+    description: Collecting [stubstatus metrics](http://nginx.org/en/docs/http/ngx_http_stub_status_module.html) from the nginx service.
```

Comment:
Is allowing markdown on all `description` fields a business rule we should implement to allow the UI to parse it correctly? That would be nice for even var descriptions too, for linking to external docs

Reply:
TBH this markdown here was coming from some copy / paste but I really like the idea of allowing markdown in the description for all description fields. Looking at your implementation it seems it is not very complex to allow this.